### PR TITLE
Add Upcoming Events section

### PR DESCRIPTION
**Pull request by barbaraliau**
Add Event Section, required for Hack Temple newsletter mention. All event mentions are links.

### CODE REVIEW

File: tokensale.html [hunk 1]
```diff
@@ -821,6 +821,64 @@ <h4 class="team-member-name">Warren Weber</h4>
...
+          </p>
+          <div class="spacer20"></div>
+        </div>
+      </div>
+      <div class="spacer20"></div>
+
+      <div class="why-use-points">
+        <div class="why-token-point">
+          <a href="https://storj.me/HackTemple" target="_blank">
+            <div class="why-token-icon" style="height: 150px; width: 150px;">
```
Comment (super3) [inline tokensale.html]: Please put in inline css in the stylesheet.

File: tokensale.html [hunk 2]
```diff
@@ -821,6 +821,64 @@ <h4 class="team-member-name">Warren Weber</h4>
...
+              May 12th <br>
+            </div>
+          </a>
+          <p>
+            Storj will be talking about the Storj Token Sale in more detail and answering questions about our future goals, technology, and users
+          </p>
+        </div>
+        <div class="why-token-point">
+          <a href="https://storj.me/Ethereal17" target="_blank">
+            <div class="why-token-icon" style="height: 150px; width: 150px;">
```
Comment (super3) [inline tokensale.html]: Please put in inline css in the stylesheet.

File: tokensale.html [hunk 3]
```diff
@@ -821,6 +821,64 @@ <h4 class="team-member-name">Warren Weber</h4>
...
+              Ethereal Summit, NYC <br>
+              May 19th <br>
+            </div>
+          </a>
+          <p>Ethereal Summit aims to highlight the technologies and innovators that are ushering in the next generation of the internet</p>
+        </div>
+        <div class="why-token-point">
+          <a href="https://storj.me/Consensus-17" target="_blank">
+            <div class="why-token-icon" style="height: 150px; width: 150px;">
```
Comment (super3) [inline tokensale.html]: Same here.

File: tokensale.html [hunk 4]
```diff
@@ -821,6 +821,64 @@ <h4 class="team-member-name">Warren Weber</h4>
...
+              Consensus, <br>NYC <br>
+              May 22-24th
+            </div>
+          </a>
+          <p>3rd annual blockchain technology summit hosted by CoinDesk. The event will feature 100+ speakers and 2000+ attendees
+          </p>
+        </div>
+        <div class="why-token-point">
+          <a href="https://storj.me/TokenSummit17" target="_blank">
+            <div class="why-token-icon" style="height: 150px; width: 150px;">
```
Comment (super3) [inline tokensale.html]: Same here.

**Comment by barbaraliau** [comment]
Changed. Ended up being unused and forgot to remove them.

**Comment by super3** [comment]
@barbaraliau Reviewed and tested. ACK.

**Pull request merged**